charts/newrelic-infrastructure/values.yaml

```diff
@@ -120,6 +134,10 @@ ksm:
               matchLabels:
                 app.kubernetes.io/name: kube-state-metrics
           weight: 100
+  # -- Config for the Infrastructure agent.
+  # Will be used by the forwarder sidecars and the agent running integrations.
+  # See: https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/
+  agentConfig: {}
```

Comment:
Ditto, only for the KSM sidecar

Reply:
Solved in [#34a64bf](https://github.com/newrelic/nri-kubernetes/pull/422/commits/34a64bfdba0341b3f159726705bcf0e54541c9f4)